python3/src/stlib3/physics/rigid/RigidObject.py

```diff
@@ -10,6 +10,7 @@ def RigidObject(name="RigidObject",
                 volume=1.,
                 inertiaMatrix=[1., 0., 0., 0., 1., 0., 0., 0., 1.],
                 color=[1., 1., 0.],
+                collisionGroup='0',
```

Comment:
I think the default behavior is "everything can collide", so 0 is not a good default value. I'll change that.